tools/tcmalloc/README.txt

```diff
@@ -0,0 +1,45 @@
+WiredTiger utilizes tcmalloc in testing through LD_PRELOAD.
+
+Implicit
+========
+
+To load tcmalloc into your current environment run:
+
+export LD_PRELOAD=$PWD/TCMALLOC_LIB/libtcmalloc.so
+
+This will be in effect until you exit the current shell.
+
+NOTE: This will affect ALL binaries run in that environment that
+dynamically link to libc.
+
+Explicit
+========
+
+Alternatively you can source the script with_tcmalloc.sh in this
```

Comment:
It looks like there's only a `define-with_tcmalloc.sh`, not `with_tcmalloc.sh` script in the directory.

Reply:
So the script defines the bash function `with_tcmalloc` so that you can run:
```
$ with_tcmalloc test/csuite/mytest
```